mesh/mesh_readers.cpp

```diff
@@ -886,215 +897,998 @@ void Mesh::ReadInlineMesh(std::istream &input, int generate_edges)
    }
 }
 
-void Mesh::ReadGmshMesh(std::istream &input)
+int Mesh::nodes_of_gmsh_element[29] =
```

Comment:
We usually use static const ints for things like these.  Sometimes it will let the compiler do a little more optimization.